### PR TITLE
k8s: add persistent-volume-claim probe

### PR DESCRIPTION
**Pull request by hunchback**
Setup using:
 
```
k8s:
  probes:
  - persistentvolumeclaim
```

Add using:

```
kubectl create -f tests/k8s/persistentvolumeclaim.yaml
```

### CODE REVIEW

**Comment by hunchback** [comment]
rerun tests

**Comment by lebauce** [comment]
@hunchback Please rebase

File: topology/probes/k8s/probe.go [hunk 1]
```diff
@@ -36,15 +36,16 @@ type Probe struct {
...
-		"service":       newServiceProbe,
+		"cluster":               newClusterProbe,
+		"container":             newContainerProbe,
+		"deployment":            newDeploymentProbe,
+		"ingress":               newIngressProbe,
+		"namespace":             newNamespaceProbe,
+		"networkpolicy":         newNetworkPolicyProbe,
+		"node":                  newNodeProbe,
+		"persistentvolumeclaim": newPersistentVolumeClaimProbe,
+		"pod":     newPodProbe,
```
Comment (lebauce) [inline topology/probes/k8s/probe.go]: please fix indentation

**Comment by hunchback** [comment]
run skydive-unit-tests

File: topology/probes/k8s/probe.go (hunk 1 above)
Comment (nplanel) [reply topology/probes/k8s/probe.go]: surprisingly if the key is 3 char ("pod") only gofmt re-indent to another way
could you move "pod" on top, for consistency

**Pull request merged**